### PR TITLE
Pathogist

### PR DESCRIPTION
**Pull request by matnguyen**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is my first PR, @bioconda/core can you confirm that things are good? Thanks!

### CODE REVIEW

File: recipes/pathogist/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: pathogist
+  version: {{ version }}
+
+source:
+  url: https://github.com/WGS-TB/PathOGiST/archive/v{{ version }}.tar.gz
+  sha256: 6691d1e08648ebe7aa743a42df51d64180824dbbd2959e449f3b11ebc39951d4
+
+build:
+  number: 1
```
Comment (dpryan79) [inline recipes/pathogist/meta.yaml]: Can you start the build number at 0?

File: recipes/pathogist/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  skip: True # [osx]
+
+requirements:
+  host:
+    - python ==3.5
```
Comment (dpryan79) [inline recipes/pathogist/meta.yaml]: Does this really only work with python 3.5? We're going to stop building for that rather soon I expect.

Comment (matnguyen) [reply recipes/pathogist/meta.yaml]: It has only been tested on Python 3.5. The next version will be tested for newer versions of Python.

**Comment by matnguyen** [comment]
@dpryan79 

The lint tests are failing due to other recipes.
I'm also not sure why the test-linux tests are failing. 

Can you clarify on what the issue may be?

**Comment by druvus** [comment]
The issue will be solved if you merge in current master into your branch.

**Comment by matnguyen** [comment]
The test-linux integration test is still failing with this error:

`cp: cannot stat '/opt/conda/conda-bld/linux-64/pathogist-0.2.2-py35_0.tar.bz2': No such file or directory`

I'm not sure how to fix this issue.

**Comment by matnguyen** [comment]
@bioconda/core

The test-linux integration test is still failing with this error:

cp: cannot stat '/opt/conda/conda-bld/linux-64/pathogist-0.2.2-py35_0.tar.bz2': No such file or directory

I'm not sure how to fix this issue.

**Pull request merged**